### PR TITLE
feat:  Ability to copy Parcel Id from Details

### PR DESCRIPTION
**Pull request by Filmaluco**
Screenshot 2023-01-16 at 19 29 16

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <string name="save">Save</string>
     <string name="send">Send</string>
     <string name="id">Id</string>
+    <string name="parcel_id">Id (parcel)</string>
```
Comment (gnarea) [inline app/src/main/res/values/strings.xml:12]: With this new field, the UI looks quite odd. I think we should rename:

- "Id" -> "Ping id".
- "Id (parcel)" -> "Parcel id".

**Review by gnarea — CHANGES_REQUESTED** [review]
Based on the screenshot you shared, this PR seems to introduce changes to the horizontal lines that separate fields. Some lines are dark grey, other light grey, and some fields are not separated by any lines.

This is what I see on my phone right now:

Screenshot_20230117-113350

I think the original design should be preserved.

**Comment by gnarea** [comment]
BTW, I believe you want to use `master` as the target branch. I didn't want to change it myself because the current target isn't in sync with `master` (same code, different commits).

**Comment by Filmaluco** [comment]
> Based on the screenshot you shared, this PR seems to introduce changes to the horizontal lines that separate fields. Some lines are dark grey, other light grey, and some fields are not separated by any lines.
> 
> This is what I see on my phone right now:
> 
> Screenshot_20230117-113350
> 
> I think the original design should be preserved.

no changes, the reason it seems like that its because its a XML preview :) 
I did against the 33 branch because we never merged it... we can now merge it, and it should be merge first.
Then this one will automatically go to master

**Comment by gnarea** [comment]
@Filmaluco, makes sense. #267 is now merged. I thought we were waiting for you to fix something in the instrumentation tests because they were failing last time I looked.

**Comment by gnarea** [comment]
BTW @Filmaluco, did you test this? Asking because of the Android Studio screenshot, so not clear if you tested it on an actual device with the gateway.

**Comment by Filmaluco** [comment]
@gnarea I tested with an actual device, did not do the test with gateway, since I was waiting on the feedback about GCP issue. will make a test on Monday at the office, at home, I only have 1 android to test.

**Comment by gnarea** [comment]
> I tested with an actual device, did not do the test with gateway  since I was waiting on the feedback about GCP issue.

Keep in mind that the GCP issue only affects the the Courier app, not any other app or library.

> will make a test on Monday at the office, at home, I only have 1 android to test.

👍🏾

**Comment by Filmaluco** [comment]
> > I tested with an actual device, did not do the test with gateway  since I was waiting on the feedback about GCP issue.
> 
> Keep in mind that the GCP issue only affects the the Courier app, not any other app or library.
> 
> > will make a test on Monday at the office, at home, I only have 1 android to test.
> 
> 👍🏾

in that case, its tested :) you can merge.

**Comment by Filmaluco** [comment]
@gnarea just finished testing this version of ping against gateway from https://github.com/relaycorp/relaynet-gateway-android/pull/675.

image

**Review by gnarea — APPROVED** [review]
Amazing, thanks!

**Pull request merged**